lib/pages/new_private_chat/widget/external_contact_widget.dart

```diff
@@ -0,0 +1,82 @@
+import 'package:fluffychat/presentation/model/presentation_contact.dart';
+import 'package:flutter/material.dart';
+import 'package:linagora_design_flutter/linagora_design_flutter.dart';
+
+import 'package:fluffychat/config/app_config.dart';
+import 'package:fluffychat/pages/search/recent_item_widget_style.dart';
+import 'package:fluffychat/widgets/avatar/avatar.dart';
+
+class ExternalContactWidget extends StatelessWidget {
+  final PresentationContact contact;
+  final void Function() onTap;
+
+  const ExternalContactWidget({
+    required this.contact,
+    required this.onTap,
+    Key? key,
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Material(
+      borderRadius: BorderRadius.circular(AppConfig.borderRadius),
+      clipBehavior: Clip.hardEdge,
+      color: Colors.transparent,
+      child: Padding(
+        padding: RecentItemStyle.paddingRecentItem,
+        child: Theme(
+          data: ThemeData(
+            splashColor: Colors.transparent,
+            highlightColor: Colors.transparent,
+          ),
+          child: ListTile(
+            contentPadding: EdgeInsets.zero,
+            title: Row(
+              crossAxisAlignment: CrossAxisAlignment.center,
+              children: [
+                SizedBox(
+                  width: RecentItemStyle.avatarSize,
+                  child: Avatar(
+                    name: contact.displayName,
+                  ),
+                ),
+                const SizedBox(width: 8),
+                Flexible(
+                  child: Column(
+                    crossAxisAlignment: CrossAxisAlignment.start,
+                    children: [
+                      Text(
+                        contact.matrixId ?? "",
+                        style: Theme.of(context).textTheme.titleMedium?.merge(
+                              TextStyle(
+                                overflow: TextOverflow.ellipsis,
+                                letterSpacing: 0.15,
+                                color: Theme.of(context)
+                                    .colorScheme
+                                    .onSurfaceVariant,
+                              ),
+                            ),
```

Comment:
Why do you use `merge`?
Currently, It was full of basic information
<img width="531" alt="Screenshot 2023-08-16 at 09 00 18" src="https://github.com/linagora/twake-on-matrix/assets/99852347/238a2c84-c847-456d-a632-03275e2df8a4">
IMO: Should use `Theme.of(context).textTheme.titleMedium?.copyWith(color: anything)` and `overflow: TextOverflow.ellipsis` outside `style`

Reply:
done